src/Extension/Workflow/TransitionBooleanResolver.php

```diff
@@ -22,6 +22,19 @@ class TransitionBooleanResolver implements TransitionResolverInterface
      */
     public const FAILED = 'failed';
 
+    /**
+     * @var string
+     */
+    protected const RESOLVER_NAME = 'boolean_resolver';
```

Comment:
you need to fix it here  if you resolve it by name
![image](https://user-images.githubusercontent.com/5063777/186104640-0797f7e3-3b41-4669-b3a1-2ae725ae90cc.png)